src/utils/rfqm_health_check.ts

```diff
@@ -97,6 +103,12 @@ export async function computeHealthCheckAsync(
         RFQM_HEALTH_CHECK_ISSUE_COUNTER.labels(issue.status, issue.label).inc(),
     );
 
+    if (gasPrice) {
+        const totalWorkerBalance = heartbeats.reduce((total, { balance }) => total.plus(balance), new BigNumber(0));
+        const totalSystemTradeCapacity = totalWorkerBalance.div(gasPrice.times(RFQM_TX_GAS_ESTIMATE));
```

Comment:
so technically, we need to:

- take the floor of the amount of tx a worker can do
- sum the floors

Otherwise, we might get into a scenario where each worker has like the ability to do 0.99 trades, and we say "this is enough for N - 1 trades", when in fact we have 0